cmake/kokkos_arch.cmake

```diff
@@ -79,7 +79,7 @@ KOKKOS_ARCH_OPTION(INTEL_XEHP      GPU  "Intel GPU Xe-HP")
 IF(KOKKOS_ENABLE_COMPILER_WARNINGS)
   SET(COMMON_WARNINGS
     "-Wall" "-Wunused-parameter" "-Wshadow" "-pedantic"
-    "-Wsign-compare" "-Wtype-limits" "-Wuninitialized")
+    "-Wsign-compare" "-Wtype-limits" "-Wuninitialized" "-Wimplicit-fallthrough")
```

Comment:
No you need to add to `GNU_WARNINGS` as we discussed earlier.  This flag is not supported by EDG.